### PR TITLE
Fix/segwit swap balance

### PR DESCRIPTION
**Pull request by TheLeoB**
## Description (optional)
1. Update imports on some files, I think were added as part of merge from refactors branch. Tests were failing because of this.
2. Some gnarly logic to change how we determine accounts balance for swap. Structure of accounts have changed to include `derivations` key, and we need to get both xPub for each account, and then add the two balances together for total. 
## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/data/coins/selectors/coins/btc.tsx [hunk 1]
```diff
@@ -37,42 +37,52 @@ export const getAccounts = createDeepEqualSelector(
...
-            balance: prop('final_balance', prop(prop('xpub', acc), btcData)),
-            baseCoin: coin,
-            coin,
-            label: prop('label', acc) || prop('xpub', acc),
-            type: ADDRESS_TYPES.ACCOUNT
-          }))
-          .filter(propEq('archived', false)))
+        // each account has a derivations object with legacy address and segwit address
+        // need to extract each xpub for balance
+        const xpubArray = acc =>
```
Comment (TheLeoB) [inline packages/blockchain-wallet-v4-frontend/src/data/coins/selectors/coins/btc.tsx:44]: gnarly logic begins here. If there's a simpler way to do this, I'd love it.

Comment (schnogz) [reply packages/blockchain-wallet-v4-frontend/src/data/coins/selectors/coins/btc.tsx:44]: there might be a generic selector for doing this. if not we should consider making one. looking now 👀

Comment (schnogz) [reply packages/blockchain-wallet-v4-frontend/src/data/coins/selectors/coins/btc.tsx:44]: actually this is fine the way it is

File: packages/blockchain-wallet-v4-frontend/src/data/coins/selectors/coins/btc.tsx [hunk 2]
```diff
@@ -37,42 +37,52 @@ export const getAccounts = createDeepEqualSelector(
...
-            archived: prop('archived', acc),
-            // @ts-ignore
-            balance: prop('final_balance', prop(prop('xpub', acc), btcData)),
-            baseCoin: coin,
-            coin,
-            label: prop('label', acc) || prop('xpub', acc),
-            type: ADDRESS_TYPES.ACCOUNT
-          }))
-          .filter(propEq('archived', false)))
+        // each account has a derivations object with legacy address and segwit address
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/data/coins/selectors/coins/btc.tsx:42]: change comment ending to "with legacy and segwit xpubs"

**Review by schnogz — CHANGES_REQUESTED** [review]

**Review by schnogz — APPROVED** [review]

**Pull request merged**